libs/shared/ui/src/lib/components/icon/icons-status/spinner.tsx

```diff
@@ -0,0 +1,32 @@
+import { type ReactNode, forwardRef } from 'react'
+import { twMerge } from 'tailwind-merge'
+import { type IconProps } from '../icon'
+
+type IconSpinnerProps = IconProps & {
+  children?: ReactNode
+}
```

Comment:
This should not be needed

Reply:
It should be necessary if we want to pass a children to Spinner component